tests/Unit/ApparentHorizons/Test_Tags.cpp

```diff
@@ -205,6 +205,43 @@ struct SomeTag : db::SimpleTag {
   using type = SomeType;
 };
 
+void test_time_deriv_strahlkorper_compute() {
+  const size_t l_max = 2;
+  Strahlkorper<Frame::Inertial> strahlkorper{l_max, l_max, 1.0,
+                                             std::array{0.0, 0.0, 0.0}};
+  SpherepackIterator iter{l_max, l_max};
+  // Set a random coefficient non-zero
+  strahlkorper.coefficients()[iter.set(2, 1)()] = 1.3;
+
+  for (size_t num_times = 2; num_times <= 4; num_times++) {
+    CAPTURE(num_times);
+    std::deque<std::pair<double, Strahlkorper<Frame::Inertial>>>
+        previous_strahlkorpers{};
+
+    // Set all strahlkorpers to be the same
+    for (size_t i = 0; i < num_times; i++) {
+      previous_strahlkorpers.emplace_front(
+          std::make_pair(static_cast<double>(i), strahlkorper));
+    }
+
+    const auto box = db::create<
+        db::AddSimpleTags<ah::Tags::PreviousStrahlkorpers<Frame::Inertial>>,
+        db::AddComputeTags<
+            ah::Tags::TimeDerivStrahlkorperCompute<Frame::Inertial>>>(
+        previous_strahlkorpers);
+
+    const DataVector& time_deriv_strahlkorper_coefs =
+        db::get<ah::Tags::TimeDerivStrahlkorper<Frame::Inertial>>(box)
+            .coefficients();
+
+    // Since we made all the Strahlkorpers the same, the time deriv should be
+    // zero.
```

Comment:
Test a case where the function actually does something.  One point with two times would be fine.